### PR TITLE
[nrf toup] Reduce Packet Buffer Pool size to 11

### PR DESCRIPTION
**Pull request by adigie**

### CODE REVIEW

File: src/platform/nrfconnect/SystemPlatformConfig.h [hunk 1]
```diff
@@ -48,7 +48,7 @@ struct ChipDeviceEvent;
 #define CHIP_SYSTEM_CONFIG_USE_SOCKETS 1
 
 #ifndef CONFIG_ARCH_POSIX
-#define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 15
+#define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 11
```
Comment (Damian-Nordic) [inline src/platform/nrfconnect/SystemPlatformConfig.h:51]: Is this selected based on the top usage in tests?

**Review by Damian-Nordic — APPROVED** [review]

File: src/platform/nrfconnect/SystemPlatformConfig.h (hunk 1 above)
Comment (Damian-Nordic) [reply src/platform/nrfconnect/SystemPlatformConfig.h:51]: Also, please fill in this info in the commit description :)

Comment (LuDuda) [reply src/platform/nrfconnect/SystemPlatformConfig.h:51]: As this is not yet targeted for 2.8.0 and we have a bit of time - what do you think to introduce a KConfig for this configuration?

Comment (adigie) [reply src/platform/nrfconnect/SystemPlatformConfig.h]: > As this is not yet targeted for 2.8.0 and we have a bit of time - what do you think to introduce a KConfig for this configuration?

Good idea. 


> Is this selected based on the top usage in tests?

Yes, highest observed pool usage was 10. I updated commit description.

File: config/nrfconnect/chip-module/Kconfig [hunk 2]
```diff
@@ -459,4 +459,10 @@ config CHIP_ENABLE_BDX_LOG_TRANSFER
 	help
 	  Enables the BDX protocol for diagnostics log transfer purposes.
 
+config CHIP_SYSTEM_PACKETBUFFER_POOL_SIZE
+  int "Packet buffer pool size"
+  default 11
+  help
+    Total number of packet buffers.
```
Comment (ArekBalysNordic) [inline config/nrfconnect/chip-module/Kconfig:466]: I wonder if we can describe it a little bit more. For example, describe the purposes for which those packet buffers will be used.

**Review by ArekBalysNordic — APPROVED** [review]
Looks good. Please squash commits before merging.

**Pull request merged**